packages/web3-utils/src/errors.ts

```diff
@@ -155,3 +154,33 @@ export class InvalidTypeAbiInput extends Web3Error {
 		super(value, 'components found but type is not tuple');
 	}
 }
+
+// TODO Type of value for all errors should be any as
+// any value can be passed when an error occurs
+export class InvalidBlockNumberOrTag extends Web3Error {
```

Comment:
Always suffix all error constructorts with `Error`, e,g, `InvalidBlockNumberOrTag` -> `InvalidBlockNumberOrTagError`. Same for other created error consstrutors.

Reply:
I agree, we have a couple other instances of class names missing `Error` suffix in other packages that we'll need to update